bokehjs/gulp/tasks/scripts.coffee

```diff
@@ -62,6 +62,7 @@ gulp.task "scripts:tsjs", ["scripts:coffee", "scripts:js", "scripts:ts"], () ->
         gutil.log("#{gutil.colors.red(real)}#{rest}")
         return
 
+      # XXX: can't enable "6133", because CS generates faulty code for closures
```

Comment:
@bryevdv, I have 0.12.4 installed. `^1.10.0` version specification in `package.json` means versions `>= 1.10.0 && <2.0.0` are allowed, so most of the time we use most recent versions of our dependencies (as long as packages are updated on regular basis).